app/src/main/res/layout/activity_quiz_submission.xml

```diff
@@ -7,7 +7,7 @@
     android:paddingLeft="@dimen/activity_horizontal_margin"
     android:paddingRight="@dimen/activity_horizontal_margin"
     android:paddingTop="@dimen/activity_vertical_margin"
-    tools:context="swen_anigans.mathmaticfanatic.QuizSubmissionActivity">
+    tools:context="swen_anigans.mathematicfanatic.QuizSubmissionActivity">

```

Comment:
Would you be able to remove the "swen_anigans.mathematicfanatic" part? I think it should still work without it.